### PR TITLE
feat: Add code examples to README

### PR DESCRIPTION
**Pull request by lxmnk**
# Checklist

- [x] I've written good commit message for all commits
- [x] I've split changes into separate commits where it's appropriate
- [ ] I've updated project version in `pyproject.toml`
- [ ] I'll make a release when PR is merged

### CODE REVIEW

**Review by Nimond — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -54,11 +77,17 @@ async def echo_handler(message: IncomingMessage, bot: Bot) -> None:
     await bot.answer_message(message.body)
 
 
+# Сюда можно добавлять свои обработчики команд
+# или переносить копировать примеры кода,
```
Comment (nidemidovich) [inline README.md:81]: Наверное, либо переносить, либо копировать.

File: README.md [hunk 2]
```diff
@@ -94,3 +129,385 @@ async def callback_handler(request: Request) -> JSONResponse:
...
+```
+
+### Сборщики обработчиков
+
+*(Этот функционал относится исключительно к `pybotx`)*
+
+```python
+from pybotx import *
+
+# Для того чтобы добавить новый обработчик команду,
```
Comment (nidemidovich) [inline README.md:259]: в команду?

File: README.md [hunk 3]
```diff
@@ -94,3 +129,385 @@ async def callback_handler(request: Request) -> JSONResponse:
...
+    # Если команда не была найдена, вызывается `default_message_handler`,
+    # если он определён. Такой обработчик может быть только один.
+    print("Hello from default handler")
+```
+
+
+### Получение системных событий
+
+*([подробное описание функции](
+https://ccsteam.atlassian.net/wiki/spaces/SMARTAPP/pages/311001185/Bot+API+BotX+API#%D0%9F%D0%BE%D0%BB%D1%83%D1%87%D0%B5%D0%BD%D0%B8%D0%B5-%D1%81%D0%BE%D0%BE%D0%B1%D1%89%D0%B5%D0%BD%D0%B8%D0%B9/%D1%81%D0%B8%D1%81%D1%82%D0%B5%D0%BC%D0%BD%D1%8B%D1%85-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9))*
```
Comment (kutuzov13) [inline README.md:186]: Would you like to shorten the link?
Example: https://bit.ly/3OF0id6
I don't think they change often.

**Review by nidemidovich — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -94,3 +128,385 @@ async def callback_handler(request: Request) -> JSONResponse:
...
+from pybotx import *
+
+collector = HandlerCollector()
+
+
+@collector.command("/bubbles", description="Send buttons")
+async def bubbles_handler(message: IncomingMessage, bot: Bot) -> None:
+    # Если вам нужна клавиатура под полем для ввода сообщения,
+    # используйте KeyboardMarkup.
+    bubbles = BubbleMarkup()
```
Comment (kutuzov13) [inline README.md:367]: May be added example with `KeyboardMarkup`?

**Review by kutuzov13 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (lxmnk) [reply README.md:186]: Good idea. Although link is only in sources, I think it's overkill here.

**Pull request merged**